baggage/baggage.go

```diff
@@ -550,3 +539,118 @@ func (b Baggage) String() string {
 	}
 	return strings.Join(members, listDelimiter)
 }
+
+// They must follow the following rules (regex syntax):
+// keyDef      = `([\x21\x23-\x27\x2A\x2B\x2D\x2E\x30-\x39\x41-\x5a\x5e-\x7a\x7c\x7e]+)`
+// valueDef    = `([\x21\x23-\x2b\x2d-\x3a\x3c-\x5B\x5D-\x7e]*)`
+// propertyRe ^(?:\s*([\x21\x23-\x27\x2A\x2B\x2D\x2E\x30-\x39\x41-\x5a\x5e-\x7a\x7c\x7e]+)\s*|\s*([\x21\x23-\x27\x2A\x2B\x2D\x2E\x30-\x39\x41-\x5a\x5e-\x7a\x7c\x7e]+)\s*=\s*([\x21\x23-\x2b\x2d-\x3a\x3c-\x5B\x5D-\x7e]*)\s*)$.
+
+func propertyFindStringSubmatch(s string) []string {
```

Comment:
How about renaming to e.g. `parseProperty`? Can we also make a regular function comment which tells what the function accepts and returns? We can reference the W3C Baggage specification instead of mentioning some regex. I think it would be even good enough to reference https://www.w3.org/TR/baggage in `doc.go`.